### ISSUE TITLE
Defining 'cls.syntax' has been deprecated

### ISSUE THREAD

**Issue by adamjohnson** [Flet/SublimeLinter-contrib-semistandard#6]
When loading SublimeLinter-contrib-semistandard, the following error can be seen in the Sublime Console:

```
SublimeLinter: ERROR:
=====================

semistandard: Defining 'cls.syntax' has no effect anymore. Use http://www.sublimelinter.com/en/stable/linter_settings.html#selector instead.
```

Other linters have run into this issue and fixed it such as SublimeLinter/SublimeLinter-rubocop#58 and markalfred/SublimeLinter-contrib-stylint#10.

**Comment by satblip** [comment]
Same issue here, created a pull request : https://github.com/Flet/SublimeLinter-contrib-semistandard/pull/7

**Comment by Flet** [comment]
Merged and tagged as 1.0.5. Thanks again!
